tests/plugins/preview/manual/previewcdn.html

```diff
@@ -0,0 +1,72 @@
+<div id="editor"></div>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<h1>Sample <span style="color: red;">document</span></h1>
+	<p>Lorem ipsum</p>
+	<div style="page-break-after:always">
+		<span style="display:none">&nbsp;</span>
+	</div>
+	<p>dolor sit amet.</p>
+</div>
+
+<script>
+	// 1. Load local editor by default with bender.
+	// 2. Remove existing instance and add script with editor from CDN.
+	// 3. Wait for the new editor to be loaded from CDN and initialize it.
+	// 4. Add icons in IE <11 from local.
+	var path = 'https://raw.githack.com/ckeditor/ckeditor4/903fa05e6bde756a7f4e618b6d24b4db21da5ab7';
```

Comment:
This PR will be merged to `master` soon so maybe we could replace (just before merging) this URL to point to `master` branch?

Still this would mean we are not testing the current local editor version which might be confusing and is not wehat we usually want to achieve with manual tests :thinking:

Reply:
I wonder couldn't we use `http://0.0.0.0:1030/apps/ckeditor/ckeditor.js` or `http://127.0.0.1:1030/apps/ckeditor/ckeditor.js` or `http://localhost:1030/apps/ckeditor/ckeditor.js` to simulate CORS (depending what is the current host used)?

From what I have checked (from `0.0.0.0`) it will give us CORS issue (so the same think we would like to simulate here I suppose):
![image](https://user-images.githubusercontent.com/1061942/127337437-2c8e4086-6a7e-4221-8383-ba19330e7f53.png)

And then we will have test using local version.

Reply:
It's look like your suggestion with simulate CORS locally works. I just need to change it a little and it should work. Thanks!